ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs

```diff
@@ -125,10 +124,8 @@ public AnalyticalFrequencyEditorVM(ComputeComponentVM defaultCurve,  EditorActio
             _StDev = DefaultCurveData.LP3StDev;
             _Skew = DefaultCurveData.LP3Skew;
             _POR = DefaultCurveData.LP3POR;
-            MyGraphicalVM = new TableWithPlotVM(new GraphicalVM("Graphical Flow Frequency","Probability","Flow"), true);
-            MyGraphicalVM.ComputeComponentVM.XLabel = "Probability";
-            MyGraphicalVM.ComputeComponentVM.YLabel = "Flow";
-            MyGraphicalVM.PlotModel.LegendPosition = LegendPosition.TopLeft;
+            GraphicalTableWithPlotVM = new TableWithPlotVM(new GraphicalVM("Graphical Flow Frequency","Probability","Flow"), true);
```

Comment:
I created a static class called StringConstants. I have started putting labels and titles in that class. This will hopefully make it easier to update things down the road. Not totally necessary, but you could move these to that class. Probability and flow are probably already in there.

Reply:
I like that. Edits made.